data-model/src/main/java/org/alfresco/repo/content/ContentStore.java

```diff
@@ -344,4 +345,38 @@ default DirectAccessUrl requestContentDirectUrl(String contentUrl, boolean attac
     {
         return Collections.emptyMap();
     }
+
+    /**
+     * Submit a request to send content to archive (offline) state.
+     * If no connector is present or connector is not supporting sending to archive, then {@code false} will be returned.
+     * Specific connector will decide which storage class/tier will be set for content.
+     * This method is experimental and subject to changes.
+     *
+     * @param contentUrl the URL of the content which is to be archived.
+     * @return true when request successful, false when unsuccessful.
+     */
+    @Experimental
+    default boolean requestSendContentToArchive(String contentUrl)
+    {
+        throw new UnsupportedOperationException("Request to archive content is not supported by this content store.");
+    }
+
+    /**
+     * Submit a request to restore content from archive (offline) state.
+     * If no connector is present or connector is not supporting restoring fom archive, then {@code false} will be returned.
```

Comment:
See earlier comment (If no connector is present ... then false will be returned).

Reply:
Rephrased javadoc to fit into real behaviour.